client-src/elements/form-field-specs.js

```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
     help_text: html`Link(s) to TAG review(s), or explanation why this is
                 not needed.`,
+    extra_help: html`
+    <p>
+      The <a target="_blank" href="https://www.w3.org/2001/tag/">W3C Technical Architecture Group</a> (TAG) 
+      is a special working group of the W3C that consists of a few appointed and elected members, 
+      all of whom are experienced members of the web standards community. 
+      The Blink launch process has a formal requirement for requesting a 
+      <a target="_blank" href="https://github.com/w3ctag/design-reviews">TAG review</a> for all features. 
```

Comment:
@miketaylr mentioned that the launch process's formal requirement is only for the design review, not the specification review. Mike, could you check that the proposed text is what we should be telling feature authors?

Reply:
Let me chat with some additional API OWNERS in our weekly meeting tomorrow - my read on https://www.chromium.org/blink/launching-features/ is that we require design reviews before shipping (generally), not design reviews + specification reviews.

Reply:
Mike posted about this at:   https://groups.google.com/a/chromium.org/g/blink-api-owners-discuss/c/py_scEqzKPE
So the answer in replies (so far) is still unknown, and it seems possible that only a design review is required.

Reply:
Once we have consensus, I'll reply here (and then go clarify the requirements).